inspec-core.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
   spec.files = %w{README.md LICENSE} + Dir.glob('{bin,lib,etc}/**/*', File::FNM_DOTMATCH)
                                           .reject { |f| File.directory?(f) || f =~ /aws|azure|gcp/ || f =~ %r{lib/plugins/.*/test/} }
 
-  spec.executables   = %w{inspec}
+  spec.executables   = []
```

Comment:
I think we can remove this line entirely from both specs unless we want to be explicit here.

Reply:
Done, thanks